cpp/modmesh/math.hpp

```diff
@@ -34,6 +107,14 @@ inline constexpr T pow(T base)
     return detail::pow(base, std::integral_constant<size_t, N>());
 }
 
+template <typename T>
+inline constexpr T pi = detail::pi_v<T>;
+
+template <typename T>
+using Complex = detail::ComplexImpl<T>;
+
+template <template <typename> typename T1, typename T2>
+inline T2 norm(const T1<T2> & input) { return input.real() * input.real() + input.imag() * input.imag(); }
```

Comment:
Why not making `norm()` a member function?

Reply:
I just want to mimic the style of the c++ stdlib.

Reply:
We don't need the C++ stdlib style for this.  The helpers may need to be exposed to Python which cannot use the C++ stdlib style.  Please make it a member function.